spec.txt

```diff
@@ -9192,6 +9199,9 @@ foo <a href="\*">
 ````````````````````````````````
 
 
+Not interpreting backslash escapes can mean not recognizing
+a tag that HTML5 would recognize:
```

Comment:
I’m not a fan of this sentence: HTML(5) definitely does recognise this. It’s a “valid” but weird link to `\` and an attribute `"` (w/o value).

Perhaps better to say something along the lines of:

```
As backslash escapes are a markdown thing, and not an HTML thing,
they are not interpreted in HTML:
```